bootstrap/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi,
```

Comment:
If you have modified the Pom you should at least update the date range (unclear if this was personal time or work time project so adding employer is your personal due diligence)